app/cni/pkg/cni/main_linux.go

```diff
@@ -208,6 +208,11 @@ func prepareResult(conf *PluginConf, logger logr.Logger) error {
 	return types.PrintResult(result, conf.CNIVersion)
 }
 
+func errorLogged(logger logr.Logger, err error, message string) error {
```

Comment:
Why is the a change in the CNI for something that's not related?

Reply:
It's related. When I was debugging this feature, I found these log entries were only in the pod descriptions, but not in kuma-cni pod logs. There are other warnings printed in kuma-cni pod logs, so they are not consistant and it confused me and costed more time to find out these logs.

Reply:
I agree it's good to make it an independent change. I made it here just because it was a quick and small change.

Reply:
Right, but the fact that you are including a bunch of "small" changes in this one PR makes it bigger which possibly explains why you are struggling to get reviews.

Reply:
Thanks for the suggestion, I'll improve the practice in future PRs.